submitit/core/job_environment.py

```diff
@@ -130,8 +130,7 @@ def _handle_signals(self, paths: JobPaths, submission: DelayedSubmission) -> Non
         @plugin-dev: Should be adapted to the signals used in this cluster.
         """
         handler = SignalHandler(self, paths, submission)
-        signal.signal(signal.SIGUSR1, handler.checkpoint_and_try_requeue)
-        signal.signal(signal.SIGTERM, handler.bypass)
+        signal.signal(signal.SIGTERM, handler.checkpoint_and_try_requeue)
```

Comment:
Can you add a signal handler for sigcont and sigusr1 (ust logging that a signal was received, nothing more? This can be helpful for future updates

Reply:
done